modules/rostests/winetests/advapi32/service.c

```diff
@@ -36,6 +36,7 @@ static const CHAR spooler[] = "Spooler"; /* Should be available on all platforms
 static CHAR selfname[MAX_PATH];
```

Comment:
Please apply this commit second, so that testbot won't time out.

Reply:
I will swap the commits.
NB: My point was to prove that the fix is still needed and helpful. Nonetheless, I am fine if you prefer this the other way round.